lib/graphql/plug/endpoint.ex

```diff
@@ -8,8 +8,7 @@ defmodule GraphQL.Plug.Endpoint do
     opts
   end
 
-  def call(%Conn{method: req_method, params: %{"query" => query}} = conn, opts)
-  when req_method in ["GET", "POST"] do
+  def call(%Conn{method: req_method, params: %{"query" => query}} = conn, opts) when req_method in ["GET", "POST"] do
```

Comment:
Is there some standard convention for this? I have aversions to ridiculously long lines... ;)